python/ray/_private/usage/usage_lib.py

```diff
@@ -223,9 +223,9 @@ def _put_library_usage(library_usage: str):
     assert _internal_kv_initialized()
     try:
         _internal_kv_put(
-            f"{usage_constant.LIBRARY_USAGE_PREFIX}{library_usage}",
-            "",
-            namespace=usage_constant.USAGE_STATS_NAMESPACE,
+            f"{usage_constant.LIBRARY_USAGE_PREFIX}{library_usage}".encode(),
```

Comment:
OOC why we need to encode this?

Reply:
We don't have to, `internal_kv.py` supports both string and bytes. But GcsClient only supports bytes.